### PR TITLE
Compatibility with python3

### PR DESCRIPTION
**Pull request by Algunenano**

### CODE REVIEW

**Comment by alrocar** [comment]
@Algunenano Any clue why front tests are failing?

**Comment by Algunenano** [comment]
No idea. Ibra was adding those tasks to paralellize the build and [asked the same question](https://app.slack.com/client/T024FUQQ8/C025932G6/thread/C025932G6-1575039768.001100?cdn_fallback=1).

**Comment by alrocar** [comment]
👍 let's wait for the fix before merging this.

**Comment by gonzaloriestra** [comment]
retest this please

**Comment by Algunenano** [comment]
retest this please

**Comment by Algunenano** [comment]
I'd say this is ok, the failing tasks are the ones that are gone already (as they were the same as Travis). Let me know if anything needs to be done prior to merging this, otherwise I'll just push it to overcome the 1 review limitation. At some point I need to get the ball rolling.

File: lib/build/tools/rails2grunt.py [hunk 1]
```diff
@@ -38,6 +38,6 @@ def normalize(f):
             else:
                 lines.append("'%s'," % (normalize(f) + "/**/*.js"))
 
-print "%s: [\n%s\n]" % (name, '\n'.join(lines)[:-1])
+print(("%s: [\n%s\n]" % (name, '\n'.join(lines)[:-1])))
```
Comment (gonzaloriestra) [inline lib/build/tools/rails2grunt.py]: Is the double parenthesis a typo?

Comment (Algunenano) [reply lib/build/tools/rails2grunt.py]: I think so, I'll rerun 2to3 to confirm.

Comment (gonzaloriestra) [reply lib/build/tools/rails2grunt.py]: Wow, many new changes... :thinking:

**Comment by Algunenano** [comment]
I've decided to run 2to3 over all .py files and not just the ones that were blocking me. I'm simply running 2to3 and removing extra includes from `shp_normalizer.py`.

**Pull request merged**